synapse/storage/databases/main/media_repository.py

```diff
@@ -149,6 +150,8 @@ def __init__(
             self._drop_media_index_without_method,
         )
 
+        self.unused_expiration_time = hs.config.media.unused_expiration_time
```

Comment:
Looks like this doesn't always exist, which is pretty annoying:

```suggestion
        if hs.config.media.can_load_media_repo:
            self.unused_expiration_time = hs.config.media.unused_expiration_time
        else:
            self.unused_expiration_time = None
```